### PR TITLE
Censor/Filter system

### PR DESCRIPTION
**Pull request by rag4214**
Will check if the user's message contains one of a pre-determined 'bad word' list. If so, adds that user to a .json file with their

    id : offenses : #,
         messages : []

and deletes their message from the server, with a message explaining that the language they use is not tolerated.

### CODE REVIEW

File: ricky.js [hunk 1]
```diff
@@ -3,12 +3,45 @@ const config = require("./config.json");
 const Discord = require('discord.js');
 const client = new Discord.Client();
 
+const censor = ['fag', 'f@g', 'retard', 'r3tard', 'ret@rd', 'r3t@rd'];
```
Comment (evanextreme) [inline ricky.js]: Any way we can have this be in a separate file?

Comment (rag4214) [reply ricky.js]: Sure, do you just want it as a .txt?

Comment (evanextreme) [reply ricky.js]: that can work. Would prefer to not have it in the repo, or have it be an easily changed environment variable

File: ricky.js [hunk 2]
```diff
@@ -3,12 +3,48 @@ const config = require("./config.json");
...
+
+if(!fs.existsSync("./censor.txt")){
+    fs.writeFileSync("./censor.txt");
+}
+
+const censor = fs.readFileSync('./censor.txt', 'utf8').split('\n');
+const offenders = require("./offenders.json");
+
 client.on('ready', () => {
     console.log("Let's get rightttttttttt into the news");
```
Comment (evanextreme) [inline ricky.js]: Let's also change this console message if we can to be more informative

**Pull request merged**